Source/WebCore/svg/SVGMatrix.idl

```diff
@@ -20,30 +20,31 @@
  * Boston, MA 02110-1301, USA.
  */
 
-// FIXME: This should be removed and made an alias to DOMRect.
+// http://www.w3.org/TR/SVG11/coords.html#InterfaceSVGMatrix
+
 [
     SkipVTableValidation,
     Exposed=Window,
     TaggedWrapper,
 ] interface SVGMatrix {
     // FIXME: these attributes should all be floats but since we implement
     // AffineTransform with doubles setting these as doubles makes more sense.
```

Comment:
Can this be removed as well?

Reply:
I think we can remove it once we move our AffineTransform to float and these bits to float. So unfortunately not at the moment. :-(

Reply:
Oh the specification has them as float? Interesting. Carry on.